contract/assembly/index.ts

```diff
@@ -5,108 +5,77 @@ import {
   logging,
   u128,
   ContractPromiseBatch,
-  PersistentSet,
-  PersistentVector,
+  ContractPromise,
 } from "near-sdk-core";
-
-type NearAccount = string; // example: user.near, user.testnet
-type ExternalAccount = string; // example: twitter/user
+import {
+  get_callback_result,
+  XCC_SUCCESS,
+  TGAS,
+  NO_DEPOSIT,
+  NearAccount,
+  ExternalAccountName,
+  AccountGlobalId,
+  SetWalletForAutoclaimCallbackArgs,
+  DeleteWalletForAutoclaimCallbackArgs,
+  ClaimTokensCallbackArgs,
+  GetCANetArgs,
+  AreConnectedArgs,
+} from "./external";
 
 //// MODELS
 
 // Common
 const OWNER_ACCOUNT_KEY = "a";
-const INIT_CONTRACT_KEY = "l";
-const ACTIVE_CONTRACT_KEY = "m";
-
-// Identity
-
-const externalByNear = new PersistentUnorderedMap<NearAccount, ExternalAccount>("b");
-const nearByExternal = new PersistentUnorderedMap<ExternalAccount, NearAccount>("c");
-const ORACLE_ACCOUNT_KEY = "d";
-const MIN_STAKE_AMOUNT_KEY = "e";
-
-// Requests
-
-@nearBindgen
-export class VerificationRequest {
-  constructor(
-    public nearAccount: NearAccount,
-    public externalAccount: ExternalAccount,
-    public isUnlink: boolean,
-    public proofUrl: string
-  ) {}
-}
+const INIT_CONTRACT_KEY = "b";
 
-const verificationRequests = new PersistentVector<VerificationRequest>("f");
-const pendingRequests = new PersistentSet<u32>("g");
-const approvedRequests = new PersistentSet<u32>("k");
+// Settings
+const CA_CONTRACT_KEY = "c";
+const NEAR_NETWORK = "m";
+const senderOrigin = "n";
```

Comment:
Rename it please. It confuses me.
senderOrigin => `SENDER_ORIGIN_KEY` for example

Reply:
`FIXED`